protocol/app/app.go

```diff
@@ -346,10 +350,20 @@ func New(
 	bApp.SetTxEncoder(txConfig.TxEncoder())
 
 	keys := storetypes.NewKVStoreKeys(
-		authtypes.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey, crisistypes.StoreKey,
-		distrtypes.StoreKey, slashingtypes.StoreKey,
-		govtypes.StoreKey, paramstypes.StoreKey, consensusparamtypes.StoreKey, upgradetypes.StoreKey, feegrant.StoreKey,
-		ibcexported.StoreKey, ibctransfertypes.StoreKey,
+		authtypes.StoreKey,
+		authzkeeper.StoreKey,
```

Comment:
weird this one is not set up in the same way... no `authztypes`?

Reply:
yeah not sure how `authz` was bootstrapped initially, but there is no `authz/types` and the key is under `keeper/keys.go` 😕

Reply:
looked into this quickly. `types` got removed in [this PR](https://github.com/cosmos/cosmos-sdk/pull/9042). still not sure why though